packages/vscode/src/utils/git.worker.ts

```diff
@@ -3,17 +3,36 @@ import { parentPort, workerData } from 'worker_threads';
 
 import { resolveSpawnOutput } from './git.util'
 
-const { gitPath, currentWorkspacePath, skipCount, limitCount } = workerData;
+const { gitPath, currentWorkspacePath, skipCount, limitCount,COMMIT_SEPARATOR,GIT_LOG_SEPARATOR } = workerData;
```

Comment:
(`,` 뒤에 띄어쓰기가 없는 걸 보니 전체적으로 prettier 적용이 안되었을수도 있을 것 같은데 확인 한번 부탁드립니닷)

`workerData`를 통해서 separator 상수를 전달하는 것과 아래처럼 바로 import해서 쓰는 것에 차이가 있는지 궁금합니다!

```suggestion
import { COMMIT_SEPARATOR, GIT_LOG_SEPARATOR } from "@githru-vscode-ext/analysis-engine";

const { gitPath, currentWorkspacePath, skipCount, limitCount } = workerData;
```

Reply:
> (`,` 뒤에 띄어쓰기가 없는 걸 보니 전체적으로 prettier 적용이 안되었을수도 있을 것 같은데 확인 한번 부탁드립니닷)
> 
> `workerData`를 통해서 separator 상수를 전달하는 것과 아래처럼 바로 import해서 쓰는 것에 차이가 있는지 궁금합니다!

prettier... 적용해보겠습니다.
import를 하지 않고 사용한 이유는 간단합니다. 해당 코드는 제가 작성한 것이 아니기 때문에, 최대한 안전하게 구현하려다 보니 그냥 `git.util.ts`에 있는 내용을 최대한 활용하려고 했습니다. 해당 부분은 추후에 고쳐보면 좋을 것 같습니다.